bitnami/airflow/templates/metrics-deployment.yaml

```diff
@@ -22,8 +22,7 @@ spec:
         app.kubernetes.io/name: {{ include "airflow.name" . }}
         app.kubernetes.io/component: metrics
         app.kubernetes.io/instance: {{ .Release.Name }}
-    spec:
-    {{- include "airflow.imagePullSecrets" . | nindent 6 }}
+    spec: {{- include "airflow.imagePullSecrets" . | nindent 6 }}
```

Comment:
Was the macro changed to cover the multiple worker use case?

Reply:
I'm not sure what this macro does or if it needs to be changed?